generators/plugin/templates/dot_gitignore

```diff
@@ -0,0 +1,37 @@
+*.rbc
```

Comment:
I wanted a consistent syntax so i could make a generic rule for yeoman. I may revisit it but prefer something to distinguish the "dot" templates.